src/install/index.js

```diff
@@ -66,7 +70,34 @@ const warmViews = (deployDoc) => {
 
         return updateIndexers(relevantTasks);
       })
-      .then(() => process.stdout.write('.'));
+      .then(() => logIndexersProgress());
+  };
+
+  // logs indexer progress in the console
+  // _design/doc  [||||||||||29%||||||||||_________________________________________________________]
+  const logIndexersProgress = () => {
+    const logProgress = indexer => {
+      // progress bar stretches to match console width.
+      // 60 is roughly the nbr of chars displayed around the bar (ddoc name + debug padding)
+      const barLength = process.stdout.columns - 60,
+            progress = `${indexer.progress}%`,
+            filledBarLength = (indexer.progress / 100 * barLength),
+            bar = progress
+              .padStart((filledBarLength + progress.length) / 2, '|')
+              .padEnd(filledBarLength, '|')
+              .padEnd(barLength, '_'),
+            ddocName = indexer.design_document.padEnd(35, ' ');
+
+      stageLog(`${ddocName}[${bar}]`);
```

Comment:
I don't think you want `stageLog`, I think you want `debug`. stageLog is supposed to be for logging a change to a stage, whereas this is more information about the progress of a stage.

The type you use also effects the displayed timings, so if we keep stageLog just for being used in the `stage` function then we know how long each stage took.

(`debug` over `info` because the idea, in theory, is that you can turn debug off and get just the minimum explanation of what's going on, each stage that's happening + any super relevant information.)